docs-source/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -159,25 +159,25 @@ The domain resource includes the element `serverPod` that is available under `sp
 `clusters` and `managedServers`. The `serverPod` element controls many details of how pods are created for server instances.
 
 The `shutdown` element of `serverPod` controls how servers will be shutdown.  This element has three properties:
-`shutdownType`, `timeoutSeconds`, and `ignoreSessions`.  The `shutdownType` property can be set to either `Graceful` 
+`shutdownType`, `timeoutSeconds`, and `ignoreSessions`.  The `shutdownType` property can be set to either `Graceful`, the default, 
 or `Forced` specifying the type of shutdown.  The `timeoutSeconds` property configures how long the server is given to 
-complete shutdown before the server is killed.  The `ignoreSessions` property when "false" allows the shutdown
-process to take longer to give time for any active sessions to complete up to the configured timeout.
+complete shutdown before the server is killed.  The `ignoreSessions` property, which is only applicable for graceful shutdown, when `false`,
+the default, allows the shutdown process to take longer to give time for any active sessions to complete up to the configured timeout.
 The operator runtime monitors this property but will not restart any server pods solely to adjust the shutdown options.
 Instead, server pods created or restarted because of another property change will be configured to shutdown, at the appropriate
 time, using the shutdown options set when the server pod is created.
 
 #### Shutdown environment variables
 
-The operator runtime configures shutdown behavior with the use of the following environment variables. Customers may
+The operator runtime configures shutdown behavior with the use of the following environment variables. Users may
 instead simply configure these environment variables directly.  When a customer configured environment variable is present,
```

Comment:
When a customer configured environment variable -> When a user-configured environment variable (change customer to user, globally)